rdf-abac-core/src/test/java/io/telicent/jena/abac/TestAssemblerABAC.java

```diff
@@ -128,11 +127,11 @@ private static void noLabelStoreDirectory(String dirName) {
         assembleBad(DIR+"abac-assembler-bad-4.ttl");
     }
 
-    private static void assembleBad(String filename) {
+    public static void assembleBad(String filename) {
         Assertions.assertThrows(AssemblerException.class, () ->assemble(filename));
     }
 
-    private static DatasetGraphABAC assemble(String filename) {
+    public static DatasetGraphABAC assemble(String filename) {
```

Comment:
These two functions only need to be package visibility.

Reply:
Good point